react/features/chat/components/native/ChatButton.js

```diff
@@ -55,13 +36,7 @@ class ChatButton extends AbstractButton<Props, *> {
      * @returns {void}
      */
     _handleClick() {
-        if (this.props._showNamePrompt) {
-            this.props._displayNameInputDialog(() => {
```

Comment:
how is the name requirement handled now?

Reply:
It is handled in the openChat action with, logic similar to what-s here